portfolio/src/main/webapp/background.html

```diff
@@ -4,27 +4,22 @@
     <meta charset="UTF-8">
     <title>Hadley Shapland</title>
     <link rel="stylesheet" href="style.css">
+    <!-- include jquery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
 
   <body>
-    <!-- SIDEBAR -->
-      <div class="sidenav">
-        <a href="index.html">Home</a>
-        <a href="background.html">Background</a>
-        <a href="experience.html">Experience</a>
-        <a href="funfacts.html">Fun Facts</a>
-    </div>
-    <!-- END SIDEBAR -->
+    <div id="commonlogic"></div>
```

Comment:
can we rename the id to something more descriptive (e.g. sidebar)?

Reply:
Done!